src/features/unified-share-modal/utils/convertData.js

```diff
@@ -458,30 +458,45 @@ export const convertCollabsResponse = (
  */
 export const convertCollabsRequest = (
     collabRequest: InviteCollaboratorsRequest,
+    collaboratorsList: collaboratorsListType | null,
 ): ContentSharingCollaborationsRequest => {
     const { emails, groupIDs, permission } = collabRequest;
     const emailArray = emails ? emails.split(',') : [];
     const groupIDArray = groupIDs ? groupIDs.split(',') : [];
+    const collabSet = new Set();
+    if (collaboratorsList) {
+        collaboratorsList.collaborators.forEach(collab => {
+            if (collab.type === COLLAB_USER_TYPE) {
```

Comment:
Does this need to also check for falsy emails, like the check we're doing for groups?